cmd/reviewdog/main.go

```diff
@@ -144,6 +145,7 @@ const (
 		$ export CI_REPO_OWNER="haya14busa" # repository owner
 		$ export CI_REPO_NAME="reviewdog" # repository name
 `
+    failOnErrorDoc = `Returns 1 as exit code if any errors/warnings found in input`
```

Comment:
Please run `go fmt`

Reply:
Done

Reply:
I see the same problem. Please have a look again.

Reply:
seems `gofmt cmd/reviewdog/main.go` fixed it, I am quite new in golang :wink: